en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -56,8 +50,9 @@ Follow this procedure to test the API revision that you deployed:
 2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
-
-3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+    
```

Comment:
- On line 35 above (`Now you have created the API and added a GET resource to it.`): We need to highlight the word `GET`  with backticks right?
- In line 41 : `Create Revision and Deploy` should be changed to `Create a  Revision and Deploy`

Reply:
Done